### DISCUSSION TITLE
I want use Closure using your invite link

### DISCUSSION THREAD

**Discussion by kokomif** [wesngu28/closurets#2] [General]
hello I want to use bot Closure using the invitation link you provided what is the next step for me to do?

about my discord server:
This is an Indonesian community server created because the openchat line was deleted, and this community is a combination of various games such as Arknight, PGR, Blue Archive and others.

https://discord.gg/rH8E9yyAzs

Thank you

**Comment by wesngu28** [comment]
Just invite it and it should work fine. When using it for Arknights the first slash command may take around 15 seconds because Heroku sleeps the API if it isn't used for thirty minutes.

**Reply by kokomif** [reply]
okay thank you
